docs/src/main/asciidoc/security-openid-connect-dev-services.adoc

```diff
@@ -406,6 +407,19 @@ This document refers to the `http://localhost:8080/q/dev-ui` Dev UI URL in sever
 If you customize `quarkus.http.root-path` or `quarkus.http.non-application-root-path` properties, then replace `q` accordingly.
 For more information, see the https://quarkus.io/blog/path-resolution-in-quarkus/[Path resolution in Quarkus] blog post.
 
+== Dev Services for OIDC
+
+When you work with Keycloak in production, <<dev-services-for-keycloak>> provides the best dev mode experience.
+For other OpenID Connect providers, it is recommended to enable the Dev Services for OIDC like in the example below:
+
+[source,properties]
+----
+quarkus.oidc.devservices.enabled=true
+----
+
+Once enabled, Quarkus starts a new server that supports most common OpenID Connect operations.
+Note, the Dev Services for OIDC are enabled by default if Docker and Podman are not available.
+
```

Comment:
Probably we should add 1-2 screenshots and explain that you can log in with alice/alice, bob/bob, or any user name and role list.

For Keycloak we didn't have to do that because it has documentation elsewhere, but for this we should, because nobody knows what it does and how it looks.

Reply:
I have added 2 screenshots and explanation how to log in as alice, bob or any custom user.

Reply:
This is much better, thanks.

In the future, we should try to find a solution for multi-tenant OIDC too. My most common OIDC demo uses 5+ public OIDC tenants, and I can't use the Dev UI in this case, it's disabled :(

Definitely not in this PR, don't worry :)